WTWSMS/history/characters/WtWSMS_erikw_characters.txt

```diff
@@ -5069,7 +5071,7 @@
 	culture = "roman"
 	female = yes
 	mother = 1008000
-	75.1.1 = {
+	65.1.1 = {
```

Comment:
Why this change?

Reply:
Because her mother died in 66.

Reply:
OK, also noticed that you explained this in ChangeLog. Sorry.